annotation-utils/src/main/java/edu/isi/vista/annotationutils/PushAnnotations.kt

```diff
@@ -0,0 +1,177 @@
+     * documents, compare with the previous commit to see if any changes have been
+     * made, and if so, push those changes to the Git repository specified by
+     * the parameter `repoToPushTo`.
+     *
+     * The program takes one argument: a file containing these parameter values:
+     *
+     * <ul>
+     *     <li> `inceptionUrl` is the base URL of your Inception server. </li>
+     *     <li> `inceptionUsername` is the user name of an Inception user with access to the
+     *          remote API. This privilege must be granted in the Inception administration interface.
+     *          </li>
+     *     <li> `inceptionPassword` is the password for the user above. Note that parameter files can
+     *          interpolate from environmental variables </li>
+     *     <li> `exportedAnnotationRoot` is the directory to download annotation to.  The
+     *     hierarchy beneath it will look like "projectName/documentName/documentName-userName.json" </li>
+     *     <li> `restoreJson`: "yes" if you would like to restore the original document text to the annotation
+     *     files; if your repository is public, please put "no"; if "yes", you must also provide
+     *     the following four parameter values:
+     *       <ul>
+     *         <li> `indexDirectory` is the location of the files produced by `indexFlatGigaword.java`
+    (https://github.com/isi-vista/nlp-util/blob/master/nlp-core-open/src/main/java/edu/isi/nlp/corpora/gigaword/IndexFlatGigaword.java) </li>
+     *         <li> `gigawordDataDirectory` is the location of the gigaword text files </li>
+     *         <li> `inputJsonDirectory` is the location of the stripped json files produced by `ExportAnnotations.kt` </li>
+     *         <li> `restoredJsonDirectory` is where the new json files will go </li>
+     *       </ul>
+     *     </li>
+     *     <li> `repoToPushTo`: the ssh url of the repository to which the annotation data
+     *     will be pushed; for ISI's curated training work, this is
+     *     git@github.com:isi-vista/curated-training-annotation.git,
+     *     but this will differ for other projects </li>
+     *     <li> `localWorkingCopyDirectory`: the location of the local repository; the repository
+     *     specified by `repoToPushTo` will be cloned to this location if it does not exist here</li>
+     * </ul>
+     *
+     * Precondition: Your ssh private key is saved in ~/.ssh/id_rsa.
+     * Postcondition: The exported annotations will be saved within a folder named `data/` in the local
+     * working directory.
+     */
+    if (argv.size != 1) {
+        throw RuntimeException("Expected a single argument: a parameter file")
+    }
+    val paramsLoader = SerifStyleParameterFileLoader.Builder()
+            .interpolateEnvironmentalVariables(true).build()
+    val params = paramsLoader.load(File(argv[0]))
+    val exportAnnotationsParamsBuilder = Parameters.builder()
```

Comment:
`Parameters` has a method whose name I don't remember which will log the contents of the parameters. This can sometimes be useful for debugging, so I recommend it for most scripts.